github_connector_odoo/models/github_repository.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016-Today: Odoo Community Association (OCA)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import urllib2
+from collections import defaultdict
+
+from openerp import models, fields, api
+
+
+class GithubRepository(models.Model):
+    _inherit = 'github.repository'
+
+    ci_id = fields.Integer(
```

Comment:
It would be nice if this were something liek `ci_id_external` - ending in `_id` makes me infer that I'm working with a recordset.